service/controller/app/key/key.go

```diff
@@ -162,6 +164,31 @@ func KubecConfigSecretNamespace(customResource v1alpha1.App) string {
 	return customResource.Spec.KubeConfig.Secret.Namespace
 }
 
+func AppLabelSelector(unique bool) labels.Selector {
```

Comment:
```suggestion
func AppVersionSelector(unique bool) labels.Selector {
```

I think this is clearer as we are selecting by version. WDYT?

Reply:
Thanks, It's clearer to understand.